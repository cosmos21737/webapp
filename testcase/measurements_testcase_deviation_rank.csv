# ケース1: 基本
ユーザー名,測定値
A,50
B,60
C,70

# ケース2: 同点
ユーザー名,測定値
A,80
B,70
C,70
D,60

# ケース3: 全員同じ
ユーザー名,測定値
A,100
B,100
C,100

# ケース4: 極端な値
ユーザー名,測定値
A,10
B,50
C,200 