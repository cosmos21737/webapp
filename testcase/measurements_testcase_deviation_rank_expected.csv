# ケース1: 基本
学年,氏名,50m走 [秒],測定日,期待順位,期待偏差値
1,A,50,2025/4/10,3,37.76
1,B,60,2025/4/10,2,50.00
1,C,70,2025/4/10,1,62.24

# ケース2: 同点
学年,氏名,50m走 [秒],測定日,期待順位,期待偏差値
1,A,80,2025/4/11,1,61.24
1,B,70,2025/4/11,2,48.76
1,C,70,2025/4/11,2,48.76
1,D,60,2025/4/11,4,41.24

# ケース3: 全員同じ
学年,氏名,50m走 [秒],測定日,期待順位,期待偏差値
1,A,100,2025/4/12,1,50.00
1,B,100,2025/4/12,1,50.00
1,C,100,2025/4/12,1,50.00

# ケース4: 極端な値
学年,氏名,50m走 [秒],測定日,期待順位,期待偏差値
1,A,10,2025/4/13,3,41.60
1,B,50,2025/4/13,2,48.00
1,C,200,2025/4/13,1,60.40 